### ISSUE TITLE
influx CLI: Display -h text when no arguments 

### ISSUE THREAD

**Issue by alsargent** [influxdata/influxdb#17936]
__Proposal:__
Display more helpful text when the user enters an `influx` command without any arguments.

__Current behavior:__
Today, when a user enters the following commands without any arguments:

```
influx config
influx ping 
influx pkg 
```

... we will typically display error text that isn't all that helpful in guiding the user on what to do.

For `influx ping`, the error text is:

```
Error: Invalid character '<' looking for beginning of value.
See 'influx ping -h' for help
```

For `influx config`, it's

```
Error: accepts 1 arg(s), received 0
See 'influx config -h' for help
```

For `influx ping`, it's

```
Error: Organization not found.
See 'influx pkg -h' for help
```

I could go on, but you get the point.

__Desired behavior:__

In each of these cases, we should explain the problem in plain English: there are no arguments to the command.

And, we should provide the same help text that `influx foo -h` provides. Why make the user type one more thing?

__Alternatives considered:__
Describe other solutions or features you considered.

__Use case:__
Why is this important (helps with prioritizing requests)?

Our company focus is on time-to-awesome for developers. Clear error messages, and saving users from having to type another command just to get help text, is one more step towards time-to-awesome.

**Comment by rbetts** [comment]
Could also ship shell completions and a man page. 

Personally - I dislike noisy commands that fill my terminal unexpectedly. Definite pet peeve when I fat-finger something and get a page of help.

**Comment by jsteenb2** [comment]
@rbetts shell completions have been included since [march](https://github.com/influxdata/influxdb/commit/7b1efe542896a122b941f89ddc02d416521328d6). 

@alsargent we purposefully don't show the full usage of the CLI when entering in no command. Many of our legacy commands are basically a noop. Take `influx bucket`, it does nothing except up the sub commands like `create`|`ls`, etc... The newer comands are making use of the base commands by plularizing the noun (stacks instead of stack ls) which equates a list call. Making the zero state useful you could say, pulling from the go philosophy.

The CLI has many, many warts. I would say making the default spit out a bunch of usage is step in the wrong direction though. Running `-h` is second nature to anyone how who works with a CLI.

**Comment by jsteenb2** [comment]
a man page would be a great add

**Comment by russorat** [comment]
We have decided not to print out the usage text automatically when there is an error in the cli.
